frontend/src/metabase/components/Header.jsx

```diff
@@ -144,7 +142,7 @@ export default class Header extends Component {
           }
           ref={this.header}
         >
-          <div className="Entity py3">
+          <div className="Entity py2">
```

Comment:
What if we converted the html tags we touch to Styled Components right away?

At least the trivial ones to start?